CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
   feature added complexity and browser issues and the use case is likely
   handled by semantic versioning and using a proper dependency.
 
+## 0.5.x
+- Set processingMode from options or first encountered context.
+- Use array representation of @container in processing.
+- Improved error detection for @container variations.
+
```

Comment:
We've been using a mostly http://keepachangelog.com/ format and have sub-headings like Added, Changed, Fixed, etc.  The "Unreleased" part they use was a hassle so I've just been adding new stuff at the top, then on release add the version+date header.  No need to fix this, I'll just update it before releasing next 0.5.x.  No idea how useful the sub-headings are, so could simplify if desired.

Reply:
Great, thanks.